version.go

```diff
@@ -19,4 +19,4 @@
 package grpc
 
 // Version is the current grpc version.
-const Version = "1.64.0-dev"
```

Comment:
Just for my knowledge, why was it suffixed with -dev before?

Reply:
Discussed offline